### PR TITLE
cleanup filing cabinets yml

### PR DESCRIPTION
**Pull request by lzk228**
## About the PR

title
add abstract BaseFilingCabinet which has most of components
 
## Why / Balance

i love cleaning ymls

## Requirements

- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

i hope no

**Changelog**

no cl

### CODE REVIEW

**Comment by slarticodefast** [comment]
Holy hell
grafik

**Comment by EmoGarbage404** [comment]
fatal force push

**Comment by slarticodefast** [comment]
That's one way to get a quick review.
grafik

**Comment by lzk228** [comment]
fixed all now, sorry for that :trollface:

**Comment by slarticodefast** [comment]
test still broken

**Comment by lzk228** [comment]
@slarticodefast test are not broken anymore

File: Resources/Prototypes/Entities/Structures/Storage/filing_cabinets.yml [hunk 1]
```diff
@@ -62,61 +58,55 @@
...
+    price: 80
   - type: Construction
     graph: FilingCabinet
+
+- type: entity
+  name: filing cabinet
+  parent: BaseFilingCabinet
+  id: filingCabinet
+  components:
+  - type: Storage
```
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Structures/Storage/filing_cabinets.yml:71]:
```suggestion
```
Any reason you are overwriting storage again?

File: Resources/Prototypes/Entities/Structures/Storage/filing_cabinets.yml [hunk 2]
```diff
@@ -185,14 +150,11 @@
 - type: entity
   parent: [filingCabinet, BaseBureaucraticStorageFill]
```
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Structures/Storage/filing_cabinets.yml:151]: grafik
They all show up as Empty if you inherit it in this order

**Review by slarticodefast — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Structures/Storage/filing_cabinets.yml (hunk 2 above)
Comment (lzk228) [reply Resources/Prototypes/Entities/Structures/Storage/filing_cabinets.yml:151]: hm... BaseBureaucraticStorageFill inherited as the second entity, so it should overwrite the first, idk, ill check

File: Resources/Prototypes/Entities/Structures/Storage/filing_cabinets.yml (hunk 1 above)
Comment (lzk228) [reply Resources/Prototypes/Entities/Structures/Storage/filing_cabinets.yml:71]: no

**Review by slarticodefast — APPROVED** [review]
Looks good now. Thanks for the cleanup!

**Pull request merged**